WORKSPACE

```diff
@@ -21,78 +21,143 @@ workspace(name = "vaticle_typedb_driver")
 # Load @vaticle_dependencies #
 ##############################
 
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
```

Comment:
If you notice, we have specific sections separated by comments like `#Load //builder/kotlin`, etc. Do the same thing for `go` after you move the `http_archive` calls and let's move these lines somewhere lower under all the other languages to have a lower chance to break their dependencies with `go` ones. 

If they have the same dependencies, but different versions, the one declared on top will be used. Don't forget to check that your rules still work after the refactoring, ofc.